### ISSUE TITLE
Clarity with warning message

### ISSUE THREAD

**Issue by CharlesCarley** [dart-lang/source_gen#490]
It might be a little obvious after the fact, but as a newcomer to dart, I think that the warning message found in [builder.dart](https://github.com/dart-lang/source_gen/blob/master/source_gen/lib/src/builder.dart#L125) could be expanded.

If it could be changed from: 
```
    log.warning('Missing "part \'$part\';".');
```
to:
```
    log.warning('The include statement "part \'$part\';" was not found.');
```

It would provide a better sense of what is going wrong.

**Comment by agent3bood** [comment]
The warning message was not clear at all, I has to google it to figure out what it meant.

**Comment by natebosch** [comment]
Fixed with #528 . Thanks @agent3bood !
